src/lib/y2partitioner/filesystems.rb

```diff
@@ -0,0 +1,44 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/filesystems/type"
+
+module Y2Partitioner
```

Comment:
I really do not like that decision about supported FS lives in partitioner and not in storage. Is there reason for it?

Reply:
Partitioner is offering a limited subset of filesystems. That list of supported filesystems is something only related to the Partitioner.

Reply:
That's something we discussed offline. The list of possible filesystems used in the Partitioner dropdowns is there basically for historic reasons. The library supports quite some file-systems that are not present in the Partitioner.

So this is not really about the libstorage-ng or `Y2Storage` capabilities, but about what has been decided to offer in the partitioner (which is a subset of all the types actually supported in `Y2Storage`).

But that needs to be better explained.